mappings/net/minecraft/block/RespawnAnchorBlock.mapping

```diff
@@ -2,6 +2,8 @@ CLASS net/minecraft/class_4969 net/minecraft/block/RespawnAnchorBlock
 	FIELD field_23153 CHARGES Lnet/minecraft/class_2758;
 	FIELD field_26442 VALID_HORIZONTAL_SPAWN_OFFSETS Lcom/google/common/collect/ImmutableList;
 	FIELD field_26443 VALID_SPAWN_OFFSETS Lcom/google/common/collect/ImmutableList;
+	FIELD field_31231 NO_CHARGES I
```

Comment:
Is this `number of charges`? or what does `no` mean?

If this is num of charges, then `CHARGES_COUNT` is better.

Reply:
It is no charges:
```java
public static final int field_31231 = 0;
public static final int field_31232 = 4;
```